.circleci/config.yml

```
@@ -131,10 +89,10 @@ jobs:
       - checkout
       - run:
           name: Test all
-          command: go test -a -ldflags '-s' ./pkg/... ./cmds/... ./xcmds/...
+          command: go test -a -ldflags '-s' ./pkg/... ./cmds/... ./xcmds/... .
```

Could we instead use this?

```suggestion
          command: go test -a -ldflags '-s' ./...
```

---

that'll run all tests in the vendor directory